### PR TITLE
Adding VirtualCluster CRD

### PR DESCRIPTION
**Pull request by dalbhanj**
Issue #, if available: related to https://github.com/aws-controllers-k8s/community/issues/1267

Description of changes: Create `VirtualCluster` CRD 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by jaypipes — CHANGES_REQUESTED** [review]
See inline @dalbhanj, I think you can remove a few lines from your generator.yaml file :)

File: generator.yaml [hunk 1]
```diff
@@ -2,10 +2,21 @@ ignore:
...
-  - VirtualCluster
+  # - VirtualCluster
   - ManagedEndpoint
   shape_names: null
 model_name: emr-containers
-operations: {}
+operations:
+  CreateVirtualCluster:
+  ListVirtualClusters:
+    output_wrapper_field_path: VirtualCluster
```
[the remark this answers is not in the document]
Comment (dalbhanj) [reply generator.yaml]: Gotcha! I removed ListVirtualClusters call as suggested. Thanks!

File: generator.yaml [hunk 2]
```diff
@@ -8,8 +8,6 @@ ignore:
 model_name: emr-containers
 operations:
   CreateVirtualCluster:
```
Comment (jaypipes) [inline generator.yaml:10]: You can remove `CreateVirtualCluster` above as well :)

**Review by jaypipes — CHANGES_REQUESTED** [review]

**Review by jaypipes — APPROVED** [review]
rock on :)

**Pull request merged**